### PR TITLE
DEV-61 ui-fixes for tablet inner pages

### PR DESCRIPTION
**Pull request by mharoonm**

### CODE REVIEW

File: src/app/mail/mail-list/mail-folder/generic-folder/generic-folder.component.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
         <li class="btn-dropdown mailbox-dropdown checkbox-dropdown">
           <div ngbDropdown>
             <button ngbDropdownToggle class="dropdown-toggle" role="button" id="actionsDropdownMenuButton"
-                    placement="top" [ngbTooltip]="'mail_list.select_messages' | translate">
+                    placement="top">
```
[the remark this answers is not in the document]
Comment (mharoonm) [reply src/app/mail/mail-list/mail-folder/generic-folder/generic-folder.component.html]: Please check now.  https://github.com/CTemplar/webclient/pull/266/commits/769745e71d90d730fe481f98e2aab7fc9c05e7e5

File: src/app/mail/mail-list/mail-folder/generic-folder/generic-folder.component.html [hunk 2]
```diff
@@ -25,30 +25,25 @@
           </div>
         </li>
         <li class="px-0">
-          <a (click)="refresh(true)" ngbTooltip="Refresh" placement="top" class="border-right-0 border-radius-0">
+          <a (click)="refresh(true)" placement="top" class="border-right-0 border-radius-0">
```
[the remark this answers is not in the document]
Comment (mharoonm) [reply src/app/mail/mail-list/mail-folder/generic-folder/generic-folder.component.html]: Please check now.  https://github.com/CTemplar/webclient/pull/266/commits/769745e71d90d730fe481f98e2aab7fc9c05e7e5

File: src/app/mail/mail-list/mail-folder/generic-folder/generic-folder.component.html [hunk 3]
```diff
@@ -25,30 +25,25 @@
...
             <i class="icon-self-timer"></i>
           </a>
         </li>
         <li class="px-0">
           <a class="border-right-0 border-radius-right-0" placement="top"
-             (click)="moveToFolder(mailFolderTypes.ARCHIVE)"
-             [ngbTooltip]="'mail_list.move_to_archive' | translate"
-             [ngClass]="{'disabled': disableFolderActions}">
+             (click)="moveToFolder(mailFolderTypes.ARCHIVE)" [ngClass]="{'disabled': disableFolderActions}">
```
[the remark this answers is not in the document]
Comment (mharoonm) [reply src/app/mail/mail-list/mail-folder/generic-folder/generic-folder.component.html]: Please check now.  https://github.com/CTemplar/webclient/pull/266/commits/769745e71d90d730fe481f98e2aab7fc9c05e7e5

File: src/app/mail/mail-settings/mail-settings.component.html [hunk 4]
```diff
@@ -281,7 +281,7 @@ <h5 class="ui-header-subtitle text-dark mb-0"><strong translate="settings.genera
                             </div>
 
                             <!-- Will be used later -->
-                            <div class="form-content-row" *ngIf="false">
+                            <div class="form-content-row">
```
[the remark this answers is not in the document]
Comment (mharoonm) [reply src/app/mail/mail-settings/mail-settings.component.html]: Reverted in following commit
https://github.com/CTemplar/webclient/pull/266/commits/769745e71d90d730fe481f98e2aab7fc9c05e7e5

File: src/app/mail/mail-list/mail-folder/generic-folder/generic-folder.component.html [hunk 5]
```diff
@@ -7,7 +7,7 @@
         <li class="btn-dropdown mailbox-dropdown checkbox-dropdown">
           <div ngbDropdown>
             <button ngbDropdownToggle class="dropdown-toggle" role="button" id="actionsDropdownMenuButton"
-                    placement="top" [ngbTooltip]="'mail_list.select_messages' | translate">
+                    placement="top" placement="top" [ngbTooltip]="'mail_list.select_messages' | translate">
```
[the remark this answers is not in the document]
Comment (atifsaddique211f) [reply src/app/mail/mail-list/mail-folder/generic-folder/generic-folder.component.html]: Haroon, please fix it in next PR, we need to go live today, so I am merging this one.

**Pull request merged**